src/util-inl.h

```diff
@@ -491,6 +492,15 @@ void ArrayBufferViewContents<T, S>::Read(v8::Local<v8::ArrayBufferView> abv) {
   }
 }
 
+inline void CheckSafePositiveJsInt(v8::Local<v8::Value> v) {
+  CHECK(v->IsNumber());
```

Comment:
Should this function return a `v8::Maybe` instead of just crashing the code? Or maybe return a `bool`?

I'm just looking at it, and it's not very obvious that this will crash node.
Furthermore, if it's not a safe positive integer, I guess we should just inform the caller and they could appropriately handle the situation, well... situationally.

Just seems to be a more generalized approach to me, but I'm happy as long as it work anyway.

Reply:
Good idea; changed.